app/frontend/components/domains/jurisdictions/jurisdiction-screen.tsx

```diff
@@ -72,7 +72,7 @@ export const JurisdictionScreen = observer(() => {
 
   return (
     <Flex as="main" direction="column" w="full" bg="greys.white">
-      <BlueTitleBar title={qualifiedName} imageSrc={"/images/jurisdiction-bus.svg"} />
+      <BlueTitleBar title={qualifiedName} />
```

Comment:
no more bus image? :'[

Reply:
Its gonna be a house now, because housing